src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -31,8 +31,7 @@
  *
  * <p>
  *     The tool takes multiple normal sample callsets produced by {@link Mutect2}'s tumor-only mode and collates sites present in two or more samples
- *     into a sites-only VCF. The PoN captures common artifactual and germline variant sites.
- *     Mutect2 then uses the PoN to filter variants at the site-level.
+ *     into a sites-only VCF. The PoN captures common artifacts.  Mutect2 then uses the PoN to filter variants at the site-level.
```

Comment:
thanks for the update!  Can I suggest being explicit about specifically not intending to capture germline variants?  I'm pretty sure that the code and default settings indicate that if the probability(germline) > 0.5 then it's excluded from the PoN.  This probability is configurable by CLI parameter.  Similarly, the number of samples required is configurable by a CLI parameter.  I only discovered this after digging deeply, as it's not advertised well elsewhere.

Reply:
done